.eslintcache

```diff
@@ -0,0 +1 @@
range":"930","text":"878"},{"range":"930","text":"879"},{"range":"931","text":"878"},{"range":"931","text":"879"},{"range":"932","text":"878"},{"range":"932","text":"879"},{"range":"933","text":"878"},{"range":"933","text":"879"},{"range":"934","text":"878"},{"range":"934","text":"879"},{"range":"935","text":"878"},{"range":"935","text":"879"},{"range":"936","text":"878"},{"range":"936","text":"879"},{"range":"937","text":"878"},{"range":"937","text":"879"},{"range":"938","text":"878"},{"range":"938","text":"879"},{"range":"939","text":"878"},{"range":"939","text":"879"},{"range":"940","text":"878"},{"range":"940","text":"879"},{"range":"941","text":"878"},{"range":"941","text":"879"},{"range":"942","text":"878"},{"range":"942","text":"879"},{"range":"943","text":"878"},{"range":"943","text":"879"},{"range":"944","text":"878"},{"range":"944","text":"879"},{"range":"945","text":"878"},{"range":"945","text":"879"},{"range":"946","text":"878"},{"range":"946","text":"879"},{"range":"947","text":"878"},{"range":"947","text":"879"},{"range":"948","text":"878"},{"range":"948","text":"879"},{"range":"949","text":"878"},{"range":"949","text":"879"},{"range":"950","text":"878"},{"range":"950","text":"879"},{"range":"951","text":"878"},{"range":"951","text":"879"},{"range":"952","text":"878"},{"range":"952","text":"879"},{"range":"953","text":"878"},{"range":"953","text":"879"},{"range":"954","text":"878"},{"range":"954","text":"879"},{"range":"955","text":"878"},{"range":"955","text":"879"},{"range":"956","text":"878"},{"range":"956","text":"879"},{"range":"957","text":"878"},{"range":"957","text":"879"},{"range":"958","text":"878"},{"range":"958","text":"879"},{"range":"959","text":"878"},{"range":"959","text":"879"},{"range":"960","text":"878"},{"range":"960","text":"879"},{"range":"961","text":"878"},{"range":"961","text":"879"},{"range":"962","text":"878"},{"range":"962","text":"879"},{"range":"963","text":"878"},{"range":"963","text":"879"},[2126,2129],"unknown","never",[1970,1973],[1795,1798],[1835,1838],[2077,2080],[2564,2567],[2924,2927],[3391,3394],[1369,1372],[1516,1519],[1897,1900],[30,33],[27,30],[30,33],[27,30],[1798,1801],[2375,2378],[1661,1664],[1949,1952],[3953,3956],[4009,4012],[4081,4084],[4572,4575],[4628,4631],[4700,4703],[5798,5801],[1166,1169],[1186,1189],[1205,1208],[1237,1240],[1355,1358],[3683,3686],[2473,2476],[2591,2594],[4497,4500],[1524,1527],[1205,1208],[1705,1708],[2375,2378],[1148,1151],[1618,1621],[5197,5200],[5228,5231],[5236,5239],[5792,5795],[8120,8123],[8154,8157],[9896,9899],[9930,9933],[10724,10727],[10758,10761],[11529,11532],[11563,11566],[1482,1485],[1661,1664],[1724,1727],[7749,7752],[1198,1201],[1217,1220],[1224,1227],[1314,1317],[1355,1358],[1855,1858],[1867,1870],[1878,1881],[2028,2031],[4084,4087],[4617,4620],[4758,4761],[5360,5363],[5950,5953],[6613,6616],[7089,7092],[7150,7153],[7915,7918],[7962,7965],[8021,8024],[8250,8253],[8536,8539],[8567,8570],[8594,8597],[8625,8628],[8673,8676],[8706,8709],[2099,2102]]
```

Comment:
I don't think we want to add this to github

Reply:
Added to gitignore